### ISSUE TITLE
DataTable - resize control keyboard accessible

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7658]
### Is this something you or your team is interested in contributing a pull request towards?

None

### Expected Behavior

Related information in this ticket: https://github.com/grommet/grommet/issues/7273

The column re-sizer should be keyboard accessible

Relevant for WCAG rule 2.1.1
We should update the DS DataTable docs once this is resolved

**Comment by jcfilben** [comment]
PR merged and DS site updated
